### DISCUSSION TITLE
Certain ScummVM Games Not Being Imported

### DISCUSSION THREAD

**Discussion by ghost** [OnionUI/Onion#1649] [Support] [answered]
Reference: https://wiki.scummvm.org/index.php?title=Where_to_get_the_games
Example Games: The Crimson Crown; Transylvania

I retrieved the required files for the two floppy games above and added them to the SCUMMVM folder under Roms, as-is:
![image](https://github.com/user-attachments/assets/63dcba77-7b47-4855-a198-229d520d5b4f)
![image](https://github.com/user-attachments/assets/091104bd-cd9d-43de-aa26-79a55d032f42)

I loaded them to the Miyoo Mini Plus and used the **Import Games** option, selecting to use the Database name for each. However, neither of them showed up & no Shortcuts were generated for them:
![image](https://github.com/user-attachments/assets/77874941-fd46-42ad-931d-f9fdd2b5bb35)

Do I need to extract the .DSK files? Also, I'm wondering exactly how **Import Games** and **Shortcuts** work. 

**Import Games** will add freeware games that I didn't provide files/folders for. So if, for example, I provided the files for Dragon History (Draci Historie), is ScummVM detecting and loading those files, or is it instead importing and loading this game from somewhere else? As for the **Shortcuts**, there was a reference in the FAQ page for ScummVM to how you needed to create Shortcuts manually in previous OnionOS versions. Can I still do that, and if so, how?

Thanks for your time everyone.

**Comment by hotgar** [comment, marked as answer]
For both games you mentioned the only versions supported by scummvm are the DOS ones and it looks like you are trying to import the Apple ones.
[https://scummvm.org/compatibility/DEV/glk:crimsoncrown/](https://scummvm.org/compatibility/DEV/glk:crimsoncrown/)
[https://scummvm.org/compatibility/DEV/glk:transylvania/](https://scummvm.org/compatibility/DEV/glk:transylvania/)

Importing games does a recursive scan of the `Roms/SCUMMVM` directory by running [scummvm command line interface](https://docs.scummvm.org/en/v2.8.0/advanced_topics/command_line.html) which adds paths to every game `scummvm` itself recognized as supported to its own configuration file `BIOS/scummvm.ini`, i.e. a successful import creates a file with a list of games it found on your sdcard. 

Shortcut creation script basically looks through that configuration file and creates a separate shortcut file for each entry in the list. 
So if the import step failed in the first place creating a shortcut manually won't help.
